pkgs/development/interpreters/php/fix-tokenizer-php81.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/ext/tokenizer/Makefile.frag b/ext/tokenizer/Makefile.frag
+index 745cbfce91..7970027607 100644
+--- a/ext/tokenizer/Makefile.frag
++++ b/ext/tokenizer/Makefile.frag
+@@ -1,5 +1,4 @@
+-$(top_srcdir)/ext/tokenizer/tokenizer_data.c: $(top_srcdir)/Zend/zend_language_parser.y
```

Comment:
I would prefer replacing `$(top_srcdir)` by `$(top_srcdir)/../..`. Removing the dependencies should not cause issues at the moment since the files are part of the source code (i.e. not generated) but this would set a bad precedent for any generated files that are added in the future.

Reply:
Could you please review once again? Patch from @stephank has been updated.

Reply:
Looks okay now. Did not test.

Reply:
Works well for me locally at least :)